x-pack/plugins/siem/public/pages/overview/overview.tsx

```diff
@@ -22,22 +24,30 @@ import * as i18n from './translations';
 
 const basePath = chrome.getBasePath();
 
+const dateEnd = Date.now();
+const dateRange = moment.duration(24, 'hours').asMilliseconds();
+const dateStart = dateEnd - dateRange;
```

Comment:
Just a heads up that this `dateRange` will only update on a full page refresh. So if the user sits on the Host Page for an hour, then navigates to the Overview Page, the query will execute with the same dateRange that was calculate on initial page load, and thus show the same results.

If this is intentional, we may want to update the overview components to use a `cache-first` fetchPolicy since it'll be displaying the same results anyway.

Reply:
@spong, is there a way to trigger an update each time the user navigates to the overview page?

Reply:
Yeah, moving these declarations into the `OverviewComponent` will ensure they're recalculated for each render.

So something like this:

``` ts
export const OverviewComponent = pure(() => {
  const dateEnd = Date.now();
  const dateRange = moment.duration(24, 'hours').asMilliseconds();
  const dateStart = dateEnd - dateRange;

  return (
    <WithSource sourceId="default" indexTypes={indexTypes}>
      {({ auditbeatIndicesExist, filebeatIndicesExist }) =>
        indicesExistOrDataTemporarilyUnavailable(auditbeatIndicesExist) &&
        indicesExistOrDataTemporarilyUnavailable(filebeatIndicesExist) ? (
          <>
            <HeaderPage subtitle={i18n.PAGE_SUBTITLE} title={i18n.PAGE_TITLE} />

            <GlobalTime>
              {({ setQuery }) => (
                <EuiFlexGroup>
                  <Summary />
                  <OverviewHost endDate={dateEnd} startDate={dateStart} setQuery={setQuery} />
                  <OverviewNetwork endDate={dateEnd} startDate={dateStart} setQuery={setQuery} />
                </EuiFlexGroup>
              )}
            </GlobalTime>
          </>
        ) : (
          <EmptyPage
            title={i18n.NO_FILEBEAT_INDICES}
            message={i18n.LETS_ADD_SOME}
            actionLabel={i18n.SETUP_INSTRUCTIONS}
            actionUrl={`${basePath}/app/kibana#/home/tutorial_directory/security`}
          />
        )
      }
    </WithSource>
  );
});
```

Reply:
Ah, sweet! Thanks so much for that!